stix_shifter_modules/elastic_ecs/stix_translation/json/to_stix_map.json

```diff
@@ -1015,8 +1089,8 @@
         }
       ],
       "ppid": {
-        "key": "process.parent_ref.ppid",
```

Comment:
Not sure if we have `process:ppid` and `process:pgid` in STIX. May need a double check.

Reply:
Both attributes are `pid` refs. `ppid` refers to the parent process `pid`, while `pgid` refers to the process group leader `pid`. NB: `pgid` to be deprecated and referred to as `process.group_leader.pid`. 

Fixed the refs in `to_stix_map`.  See new [commit](https://github.com/opencybersecurityalliance/stix-shifter/pull/1471/commits/a301c5d30f35b382aaeb7f5e1a1257918fdadac1).

A new `group_leader_ref` reference attribute should be added to [Process STIX extensions](https://github.com/opencybersecurityalliance/stix-extensions/blob/main/2.0/custom-process.md).